src/dialog.h

```diff
@@ -55,6 +55,10 @@ typedef struct DialogTextBuffer {
 	float opacity;
 } DialogTextBuffer;
 
+struct title_box {
+	float opacity;
+};
```

Comment:
Since this is in a public header, struct definitions are generally held up to a higher standard. The name should be CamelCase and prefixed, e.g. `DialogTitleBox`, and also have a typedef with the same name.

In this specific case however, the definition is probably not even necessary: you could just define an anonymous struct inside Dialog to achieve the same effect without namespace pollution (see next comment).